src/lib/jsonParseV2/parseEntityVector.cpp

```diff
@@ -53,11 +53,16 @@ std::string parseEntityVector(ConnectionInfo* ciP, const Value::ConstMemberItera
 
   for (Value::ConstValueIterator iter2 = iter->value.Begin(); iter2 != iter->value.End(); ++iter2)
   {
+
     std::string  r;
     Entity*      eP = new Entity();
 
     evP->vec.push_back(eP);
 
+    // FIXME PR: is the logic of this function correct? I mean, parseEntityObject() could result in error, so the push_back maybe
+    // should be done after the parseEntityObject() call (and delete the eP just before return r). Or maybe it doesn't matter... as in the
+    // case of error the error is propagated back to result in an OrionError and the evP is not actually used (and properly destroyed during
+    // the release step)
     r = parseEntityObject(ciP, iter2, eP, attributesAllowed);
```

Comment:
It looks good to me, a simple comment remarking "it doesn't matter" could be enough. In fact when one entity fails to be parsed, it can be N previous parsed entities in the vector in that moment and they will be deleted when the error is returned and the whole vector is discarded (or so I think). So an empty entity does not make bad, al most a little performance improvement